packages/cli/src/util/dev/builder-cache.ts

```diff
@@ -126,31 +120,6 @@ export function filterPackage(
     return false;
   }
 
-  // Skip install of already installed Runtime with tag compatible match
-  if (
-    parsed.name &&
-    parsed.type === 'tag' &&
-    parsed.fetchSpec === distTag &&
-    buildersPkg.dependencies
-  ) {
-    const parsedInstalled = npa(
-      `${parsed.name}@${buildersPkg.dependencies[parsed.name]}`
-    );
-    if (parsedInstalled.type !== 'version') {
-      return true;
-    }
-    const semverInstalled = semver.parse(parsedInstalled.rawSpec);
-    if (!semverInstalled) {
-      return true;
-    }
-    if (semverInstalled.prerelease.length > 0) {
-      return semverInstalled.prerelease[0] !== distTag;
-    }
-    if (distTag === 'latest') {
-      return false;
-    }
```

Comment:
After removing the two last cases here, the rest could only return true, which means they did nothing. Removed this whole section as a result.